bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -66,6 +72,34 @@ pub struct KeyRequestPair {
     pub key_request: Request,
 }
 
+/// The result of a signature check of a signed JSON object.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct SignatureCheckResult {
+    /// The result of the signature check using the public key of our own
+    /// device.
+    pub device_signature: SignatureState,
+    /// The result of the signature check using the public key of our own
+    /// user identity.
+    pub user_identity_signature: SignatureState,
+    /// The result of signature checks using public keys of other devices we
+    /// own.
+    pub other_signatures: HashMap<String, SignatureState>,
```

Comment:
Optional: should this then be called `other_devices_signatures` (clients do not see the rust documentation right away)?

Reply:
Yup make sense https://github.com/matrix-org/matrix-rust-sdk/pull/1217/commits/91bbaa09dc4947ed7a29b34e0540807212aa0fe9.